File: Data Structures & Algorithms/climbing-stairs/submission-0.cpp
class Solution {
public:
    int my(int i)
    {
        if(i==0)return 1;
        if(i==1)return 1;
        if(i<0)return 0;
        int a=my(i-1);
        int b=my(i-2);
        return a+b;
    }
    int climbStairs(int n) {
        return my(n);
    }
};

File: Data Structures & Algorithms/validate-parentheses/submission-0.py
class Solution:
    def isValid(self, s: str) -> bool:
        stack = []

        for q in s:

            if q in "([{":
                stack.append(q)

            else:
                if not stack:
                    return False

                if (
                    (stack[-1] == "{" and q == "}") or
                    (stack[-1] == "[" and q == "]") or
                    (stack[-1] == "(" and q == ")")
                ):
                    stack.pop()

                else:
                    return False

        return len(stack) == 0